src/OMSimulatorLib/Component.h

```diff
@@ -92,6 +92,8 @@ namespace oms
     virtual oms_status_enu_t setRealInputDerivative(const ComRef& cref, const SignalDerivative& der) { return logError_NotImplemented; }
     virtual oms_status_enu_t stepUntil(double stopTime) { return oms_status_ok; }
     virtual oms_status_enu_t addResources(std::string& filename) { return oms_status_ok; }
+    virtual oms_status_enu_t deleteReferencesInSSD(const std::string& filename) {return oms_status_ok;}
+    virtual oms_status_enu_t deleteResourcesInSSP(const std::string& filename) {return oms_status_ok;}
```

Comment:
Should we return oms_status_error and a message that these functions aren't implemented?

Reply:
but these functions are implemented in `ComponentFMUCS.cpp` and `ComponentFMUME.cpp`

Reply:
this is fixed with this commit https://github.com/OpenModelica/OMSimulator/pull/1079/commits/4b004e3227d4e44b7804db1f8a8010131cde74d2